src/main/java/atmosfears/AtmosFearsBE/controller/TimeSeriesController.java

```diff
@@ -0,0 +1,108 @@
+package atmosfears.AtmosFearsBE.controller;
+
+import atmosfears.AtmosFearsBE.database.Particulate;
+import atmosfears.AtmosFearsBE.database.SensorCode;
+import atmosfears.AtmosFearsBE.model.AggregatedParticulates;
+import atmosfears.AtmosFearsBE.model.AirParticulates;
+import atmosfears.AtmosFearsBE.service.AirParticulatesService;
+import lombok.AccessLevel;
+import lombok.experimental.FieldDefaults;
+import org.json.JSONObject;
+import org.springframework.format.annotation.DateTimeFormat;
+import org.springframework.web.bind.annotation.*;
+
+import java.text.SimpleDateFormat;
+import java.util.*;
+
+@RestController
+@FieldDefaults(level = AccessLevel.PRIVATE)
+@RequestMapping("/api/timeseries")
+public class TimeSeriesController {
+
+    final AirParticulatesService airParticulatesService;
```

Comment:
```suggestion
    private final AirParticulatesService airParticulatesService;
```
We don't want to expose dependencies of this controller outside the controller class